### PR TITLE
closes #33 | retrieve tailors

### LINKED ISSUE #33: A user should be able to retrieve tailors in his/her area
When I make a `GET` request to the route `/api/tailors?longitude=xxxx&latitude=xxxx`

*Response*
```
[
  {
    "firstname": "xxxx",
    "lastname": "xxxxx",
    "phone": "xxxxx",
    "email": "xxxxx",
    "location": "xxxxx",
    "password": "xxxx",
    "outfitTypes": ["xxxxx","xxxx"]
  }
]
```

### PR DESCRIPTION
**Pull request by demiladebamgbose**
*Description of Task*
- A customer can retrieve nearby tailors within a 30km radius

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const server = express();
 const router = express.Router();
 
 // connect to the DB
-mongoose.connect(Config.database);
+mongoose.connect(Config.database)
```
Comment (bolorundurowb) [inline server.js]: Why did you remove the semicolon here?

File: src/server/controllers/Customers.js [hunk 2]
```diff
@@ -72,17 +73,59 @@ class Customers {
...
+        }
+      }
+    }).exec((err, tailors) => {
+      if (err) {
+        res.status(500).send({
+          message: 'An error occured'
+        });
+      }
+      if (tailors) {
+        res.json({
```
Comment (bolorundurowb) [inline src/server/controllers/Customers.js]: Please set the status, i.e `200` or otherwise

File: src/server/routes/TailorRoutes.js [hunk 3]
```diff
@@ -24,14 +24,19 @@ class TailorRoutes {
         Validations.validateTailorFeedback,
         Customers.giveFeedback
       );
```
Comment (bolorundurowb) [inline src/server/routes/TailorRoutes.js]: Why did you remove the line between?

File: src/server/routes/TailorRoutes.js [hunk 4]
```diff
@@ -24,14 +24,19 @@ class TailorRoutes {
...
-
     router.route('/tailors/profile')
       .put(
         Authentication.checkAuthentication,
         Authorization.isTailor,
         Validations.validateTailorUpdate,
         Tailors.updateProfile
-      )
+      );
```
Comment (bolorundurowb) [inline src/server/routes/TailorRoutes.js]: A think an empty line between routes would be nice.

File: tests/server/customers.test.js [hunk 5]
```diff
@@ -96,10 +96,10 @@ describe('Customers', () => {
             collectionTime: 2,
             delivery: 5
           }).end((err, res) => {
-          res.status.should.equal(201);
-          res.body.message.should.equal('Your review has been recorded');
-          done();
-        });
+            res.status.should.equal(201);
```
Comment (bolorundurowb) [inline tests/server/customers.test.js]: Nice one with the indentation.

File: src/server/controllers/Customers.js (hunk 2 above)
Comment (demiladebamgbose) [inline src/server/controllers/Customers.js]: default is 200

File: tests/server/customers.test.js (hunk 5 above)
Comment (demiladebamgbose) [inline tests/server/customers.test.js]: @bolorundurowb changes made
